.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Open a bug report
+title: "[bug] "
+labels: bug
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of the bug or unexpected behavior.
+
+**Steps To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System Informatioon:**
+
+- OS: [e.g. Ubuntu 24.04 LTS]
+- Python Version: [e.g. Python 3.11.2]
```

Comment:
This is a nodeJS app so we should replace this with nodejs version, we also need to add browser/browser version

Reply:
done